### ISSUE TITLE
[BUG] Version 6.4.0 - Installer - Errors

### ISSUE THREAD

**Issue by bovirus** [radj307/volume-control#109]
@radj307 

Missing info about version
License text file not use all space available
![image](https://github.com/radj307/volume-control/assets/1262554/7db0fd69-18ee-4e95-b661-554f08a8093b)

Installer exe file properties
(c) charcter is wrong (coding error)
Missing info about Program Version/File version
![image](https://github.com/radj307/volume-control/assets/1262554/72fffd1e-a63d-45a4-8991-5e1384e1b1cf)

**Comment by radj307** [comment]
The problems with version info were fixed in fb0992ae7b1c3fda9575c37b8ea8ef0cee2de634 when I fixed #104 

The copyright symbol was also fixed, but may still happen on some systems.  
![image](https://github.com/radj307/volume-control/assets/1927798/02f5f594-2768-4df8-99bf-c3cd42f01b61)

Can you check if the following setting is enabled on your system: **Language** -> **Administrative language settings** -> **Change system locale** **Beta: Use Unicode UTF-8 for worldwide language support**  
![image](https://github.com/radj307/volume-control/assets/1927798/b541b8d1-7fdb-4c8c-a989-ff3cec4ffc1c)

**Comment by bovirus** [comment]
@radj307 

Please take care that I used for report for installer v. 6.4.0 (released two days ago).
It didn't run under Windows 11.

I will check this evening in Windows 10.

**Comment by radj307** [comment]
(Continuing from #106)

I was able to change the license text alignment pretty easily by adding this to the `[Code]` section:
```
procedure InitializeWizard();
begin
  WizardForm.LicenseMemo.Alignment := taCenter;
end;
```

![image](https://github.com/radj307/volume-control/assets/1927798/22eed32c-001b-4b86-94d0-b0574170cc4e)

Of course, some lines are too far to the right since they are indented, but this can be fixed by stripping preceding whitespace from them as well.

**Comment by radj307** [comment]
With whitespace removed: 
![image](https://github.com/radj307/volume-control/assets/1927798/d81accd6-0326-4687-b5f6-b29733d8ac61)

**Comment by bovirus** [comment]
@radj307 

Thanks.
Please fix other problems.
LICENSE text is the last s importance.

**Comment by radj307** [comment]
All of the other problems were already solved in fb0992ae7b1c3fda9575c37b8ea8ef0cee2de634 and are available as build artifacts until the release of `v6.5.0`

If you want to download new versions before they're released, you can download the build artifact for it from the **Actions** tab at the top of this page.  

To filter the workflow runs, click on **Generate Release** in the left pane, then click on the latest one that successfully completed:  
![image](https://github.com/radj307/volume-control/assets/1927798/b0ed4fe3-ae71-4fc1-ad8a-c30e6ea98c53)  
Then scroll down to the bottom of the page & click on `build-windows`. It'll download it as an archive, which you can extract somewhere to find `VolumeControl.exe` & `VolumeControl-Installer.exe`
![image](https://github.com/radj307/volume-control/assets/1927798/f5dc4137-2939-44e8-adf8-5be011d87fa5)

**Comment by bovirus** [comment]
@radj307 

Thanks. 
Installer issues fixed in new beta.
Please close this ticket.

Plerase remind to fix strings issues
https://github.com/radj307/volume-control/issues/110
